plugins/mac-videotoolbox/encoder.c

```diff
@@ -567,6 +574,20 @@ static OSStatus create_encoder(struct vt_encoder *enc)
 		if (code != noErr) {
 			return code;
 		}
+
+		if (__builtin_available(macOS 15.0, *)) {
+			int spatialAq = enc->spatial_aq ? kVTQPModulationLevel_Default : kVTQPModulationLevel_Disable;
+			CFNumberRef SpatialAQ = CFNumberCreate(kCFAllocatorDefault, kCFNumberIntType, &spatialAq);
+
+			code = VTSessionSetProperty(s, kVTCompressionPropertyKey_SpatialAdaptiveQPLevel, SpatialAQ);
```

Comment:
Just a style nit, but CapitalCase is used in CoreFoundation only for "class"-like objects (like `CFNumberRef`) but not for variables. Whereas for `liboos` C code snake_case is to be used, so a more correct variant would be:

```suggestion
			int spatial_aq = enc->spatial_aq ? kVTQPModulationLevel_Default : kVTQPModulationLevel_Disable;
			CFNumberRef spatialAQ = CFNumberCreate(kCFAllocatorDefault, kCFNumberIntType, &spatial_aq);

			code = VTSessionSetProperty(s, kVTCompressionPropertyKey_SpatialAdaptiveQPLevel, spatialAQ);
```